nimrod/tools/bin.py

```diff
@@ -9,5 +9,5 @@
                                 'evosuite-standalone-runtime-1.0.6.jar'])
 MOD_RANDOOP = os.sep.join([PATH, 'bin', 'modified-randoop-all-4.2.5.jar'])
 JMOCKIT = os.sep.join([PATH, 'bin', 'jmockit-1.40-marcio.1.jar'])
-RANDOOP = os.sep.join([PATH, 'bin', 'randoop-all-4.2.2.jar'])
+RANDOOP = os.sep.join([PATH, 'bin', 'randoop-all-4.2.5.jar'])
```

Comment:
This new jar file already includes the report generation implemented by you, right?

Reply:
Yes, that jar is the original randoop with the report generator for tests.